README.md

```diff
@@ -1,10 +1,10 @@
 # group-03
 
 ## What are you going to make?
-We're going to make a remote controlled car with a lot of focus being put on using a mobile application with a nice UI and a bluetooth connection. The application will allow for a camera feed, forwards and backwards driving, speed control, and also turning 360 degrees. Lastly, as a bonus feature that we will hopefully have time for, we want to try to use short voice commands to activate, stop, and control the car in different ways. 
+We're going to make a remote controlled car with a lot of focus being put on using a web-application with a nice UI and a bluetooth connection. The application will allow for a camera feed, forwards and backwards driving, speed control, and also turning 360 degrees. Lastly, as a bonus feature that we will hopefully have time for, we want to try to use short voice commands to activate, stop, and control the car in different ways. 
```

Comment:
I assume you'll have a web-application running on the Raspberry Pi. If that is so, then why do you need a bluetooth connection as well?

Reply:
That was what we were debating about, we didn't know if we would still need a Bluetooth connection for the Raspberry Pi as we haven't used the technology before, but if that's not the case we'll remove that :) Thanks!

Reply:
Well, the Bluetooth connection could be used to have a client (e.g. application running on a mobile phone) communicate directly and wirelessly with the car. Since you decided to have a Raspberry Pi on it which will be hosting a web app (and is wirelessly accessible), you should be able to control the car via a serial (wired) connection.